src/formatDate.js

```diff
@@ -1,14 +1,62 @@
 'use strict';
 
-/**
- * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
- *
- * @returns {string}
- */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateToArrow = date.split(fromFormat[fromFormat.length - 1]);
+  const formatedDate = [];
+  let day = '';
+  let month = '';
+  let year = '';
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    switch (fromFormat[i]) {
+      case 'DD':
+        day = dateToArrow[i];
+        break;
+
+      case 'MM':
+        month = dateToArrow[i];
+        break;
+
+      default:
+        year = dateToArrow[i];
+        break;
```

Comment:
Maybe it's better to use separate cases for 'YY' and 'YYYY' and add default case for other wrong values